db/migrate/20190422185437_add_location_table.rb

```diff
@@ -0,0 +1,27 @@
+class AddLocationTable < ActiveRecord::Migration[5.1]
+  def up
+    create_table :locations, force: true do |t|
+      t.string :name, default: "", null: false, comment: "Full display name, such as a complete address"
+      t.string :level_name, default: "", null: false, limit: 20, comment: "Level of specificity (country, state, subdivision, or city)"
```

Comment:
Maybe call this `specificity`? Like the metadata field we never used. :(

Reply:
I think `level_name` seems good to me, or `geo_level`. Some other candidates are `granularity` and `scale`. See https://english.stackexchange.com/questions/94398/whats-the-best-term-for-geographical-level  .

Reply:
👍 ok I think I'll keep level_name here

Reply:
Ok Mark convinced me to change this so that it doesn't imply `level`_name is parallel to `country/state/subdivision`_name. I will change to geo_level